pages/api/profiles/index.js

```diff
@@ -33,3 +34,62 @@ export async function getProfiles(options = {}) {
 
   return JSON.parse(JSON.stringify(profiles));
 }
+
+export async function getUsers(options = {}) {
+  await connectMongo();
+
+  let profiles = [];
+  const { cards } = { cards: false, ...options };
```

Comment:
I'm not sure about this. If you intend to provide a default value for the 'cards' property when it's not provided, you can also use the following syntax: `const { cards = false } = options`. In my opinion, this approach is more readable. Up to you